app/src/main/res/drawable/start_button_transparent_background.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item>
+    <layer-list>
+      <item android:right="2dp" android:top="2dp">
+        <shape>
+          <corners android:radius="3dp" />
+          <solid android:color="#5CBABABA" />
+        </shape>
+      </item>
+      <item android:bottom="2dp">
+        <shape>
+          <solid android:color="#0000003D" />
+          <corners android:radius="4dp" />
+          <padding android:bottom="10dp" android:left="10dp"
```

Comment:
Not sure if 10dp is acceptable padding, @BenHenning can finalise this.

Reply:
We should be using multiples of 4dp per material specification. Please check with Chantel what the padding should be here if the Adobe Xd mocks do not have the correct value.

Reply:
This padding was given to check shadow effect. When I removed it didn't make any difference. And shadow of 2px is provided for upper item as per the mock.